src/main/java/uk/gov/companieshouse/authcodenotification/utils/Encrypter.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.authcodenotification.utils;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+@Component
```

Comment:
It's pretty low-level stuff in here. I think some comments/Javadoc would be useful.